src/main/webapp/app/shared/import-list/import-table.component.scss

```diff
@@ -0,0 +1,13 @@
+.import-table-container {
+    height: 280px;
+    overflow-y: scroll;
+
+    table {
+        thead th {
+            position: sticky;
+            top: 0;
+            z-index: 1;
```

Comment:
Could you add a comment which element would hide this otherwise?

Reply:
What do you mean with hide otherwise? This just fixes the table header that if you scroll you can always see the header

Reply:
For my understanding we would not need to set a value for the `z-index` if no other element would hide the header?